### PR TITLE
get headerTimeout from server config

### PR DESCRIPTION
**Pull request by martinoman**
👾

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const USE_DYNAMIC_RATE_LIMITING_DEFAULT = false
 const USER_ACTIVE_TIMEOUT_DEFAULT = 30000
 const INTERVAL_DEFAULT = 10000
 const LIMIT_DEFAULT = 10
+const HEADERTIMEOUT_DEFAULT = 1000
```
Comment (helenaalinder) [inline index.js]: HEADER_TIMEOUT_DEFAULT*

Comment (helenaalinder) [inline index.js]:
```suggestion
const HEADER_TIMEOUT_DEFAULT = 1000
```

Comment (helenaalinder) [inline index.js]: HEADER_TIMEOUT_DEFAULT

Comment (martinoman) [reply index.js]: It is fixx

**Pull request merged**